qlty-cli/src/commands/plugins/disable.rs

```diff
@@ -28,14 +28,23 @@ impl ConfigDocument {
     }
 
     pub fn disable_plugin(&mut self, name: &str) -> Result<()> {
-        if let Some(plugins) = self.document["plugin"].as_array_of_tables_mut() {
-            plugins.retain(|plugin| {
-                if let Some(plugin_name) = plugin.get("name").and_then(Item::as_str) {
-                    plugin_name != name
-                } else {
-                    true // Do not modify unknown data
+        if self.document.get("plugin").is_none() {
+            bail!("No plugins found in qlty.toml");
+        }
```

Comment:
Do you see any concrete downside to removing this error check and letting it fall to the second error check? This one seems redundant at first read.

Reply:
No. I don't actually. I copied this from the `upgrade` sub-command, which follows similar logic, but will remove here (and note we can/should probably remove from there as well)